hr_expense_operating_unit/tests/test_hr_expense_operating_unit.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2016-17 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo.tests import common
+from odoo.exceptions import ValidationError, UserError
+
+
+class TestHrExpenseOperatingUnit(common.TransactionCase):
+
+    def setUp(self):
+        super(TestHrExpenseOperatingUnit, self).setUp()
+        self.res_users_model = self.env['res.users']
+        self.hr_expense_model = self.env['hr.expense']
+        self.hr_expense_sheet_model = self.env['hr.expense.sheet']
+        self.hr_employee_model = self.env['hr.employee']
+
+        self.company = self.env.ref('base.main_company')
+        self.partner1 = self.env.ref('base.res_partner_1')
+        self.partner2 = self.env.ref('base.res_partner_2')
+
+        # Expense Product
+        self.product1 = self.env.ref('hr_expense.air_ticket')
+
+        self.grp_hr_user = self.env.ref('hr.group_hr_user')
+        self.grp_accou_mng = self.env.ref('account.group_account_manager')
+        self.grp_account_invoice =\
+            self.env.ref('account.group_account_invoice')
+
+        # Main Operating Unit
+        self.ou1 = self.env.ref('operating_unit.main_operating_unit')
+        # B2C Operating Unit
+        self.b2c = self.env.ref('operating_unit.b2c_operating_unit')
+
+        self.user1 = self._create_user('Test HR User 1', 'user_1', 'demo1',
+                                       [self.grp_hr_user, self.grp_accou_mng,
+                                        self.grp_account_invoice],
+                                       self.company,
+                                       [self.ou1, self.b2c])
+        self.user2 = self._create_user('Test HR User 2', 'user_2', 'demo2',
+                                       [self.grp_hr_user, self.grp_accou_mng,
+                                        self.grp_account_invoice],
+                                       self.company,
+                                       [self.b2c])
+
+        self.emp = self._create_hr_employee()
+
+        self.hr_expense1 = self._create_hr_expense(
+            self.ou1, self.emp)
+
+        self.hr_expense2 = self._create_hr_expense(
+            self.b2c, self.emp)
+
+        self._post_journal_entries(self.hr_expense1)
```

Comment:
This is ok. However, I miss a test that check the operating units for the journal items created.